packages/payments-plugin/src/mollie/mollie.service.ts

```diff
@@ -297,14 +303,14 @@ export class MollieService {
                 return;
             }
         }
-        const statesThatRequireAction: OrderState[] = [
+        const vendureStatesThatRequireAction: OrderState[] = [
             'AddingItems',
             'ArrangingPayment',
             'ArrangingAdditionalPayment',
             'PaymentAuthorized',
             'Draft',
         ];
-        if (!statesThatRequireAction.includes(order.state)) {
+        if (!vendureStatesThatRequireAction.includes(order.state)) {
```

Comment:
Refactor, because we now have `mollie states that require action` and `vendure states that require action`